src/emcpy/plots/tools.py

```diff
@@ -0,0 +1,94 @@
+class ColorbarPreset:
+    """
+    Class that has pre-determined vmin, vmax,
+    and cmap values depending on the variable
+    and evaluation type.
+    """
+    def __init__(self, variable, eval_type):
+        """
+        Class constructor.
+
+        Args:
+            variable : (str) variable of interest
+            eval_type : (str) evaluation type
+        """
+
+        cbar_d = {
+            'brightness temperature': {
+                'diff': {
+                    'vmax': -5,
+                    'vmin': 5,
+                    'cmap': 'coolwarm'
+                },
+                'magnitude': {
+                    'vmax': 320,
+                    'vmin': 200,
+                    'cmap': 'rainbow'
+                }
+            },
+            'specific humidity': {
+                'diff': {
+                    'vmin': 0.005,
+                    'vmax': 0.005,
```

Comment:
vmin=vmax?

Reply:
So I don't like how all this is defined inside of `__init__`.
The `__init__` method should provide defaults and everyone else should inherit from `ColorbarPreset` and overwrite for their specific variable.

Reply:
So you want a method to update the defaults based off of the variable?
Like this:
```
mycbar = ColorbarPreset()
mycbar.defaults('temperature', 'diff')
```
?

Reply:
Maybe something like this?
```
scatterobj = Scatter(lat, lon, data)
ColorbarPreset(scatterobj, variable='temperature', eval_type='diff')
```
the constructor would then look like this:
```
def __init__(self, plotobj, variable, eval_type):
    self._get_cbar_type(variable, eval_type)

    plotobj.vmin = self.vmin
    plotobj.vmax = self.vmax
    plotobj.cbar = self.cbar
```
with `_get_cbar_type()` being a method with all of this information in it?

Reply:
why just the colorbar type?
Why not use this to get entire field specs.  min, max, contour interval, contour labels, units, etc.

Reply:
I am thinking something along these lines:

```
class VariableSpecs:
    def __init__(self, vardict=dict())
        self.variable_name = vardict.get('name', 'VariableName')
        self.variable_sname = vardict.get('short_name', 'VName')
        self.variable_type = vardict.get('type', 'VariableType')
        self.variable_units = vardict.get('units', 'VariableUnits')

        return

class FieldSpecs:
    def __init__(self, var_obj=VariableSpecs()):
        self.range = vardict.get('range', [-1., 1.])
        self.cntr_int = vardict.get('contour_interval', 0.1)
        self.cntrs = np.arange(self.range[0], self.range[-1], self.cntr_int)

        return

class ColorbarPreset:
    def __init__(self, colobardict=dict()):

        self.eval_type = colobardict.get('eval_type', 'diff')
        _cmap = colorbardict.get('cmap', None)
        if _cmap is None:
            self.cmap = _cmap
        else:
            self.cmap = 'coolwarm' if self.eval_type in ['diff'] else 'rainbow'

        return
```
and then extending this for variables eg. `Temperature`, `Sp. Humidity`, `Ice`, etc.

Thoughts?

Reply:
So then the vardict is defined in another spot that is like this?:

```

rootvardict = {'temperature': {'range': [220, 320],
                                                  'contour_interval': 5,
                                                  'name': 'temperature',
                                                  'units': 'K',
                                                 }
                      }

def get_vardict(varname):
    return rootvardict[varname]

vardict = get_vardict('temperature')
```

I do agree we should probably try to make this more generic than just `colorbar`, for example, a time series of temperature might also want the y-axis range to be the same as the colorbar range